x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -143,15 +142,13 @@ export class AlertsClient {
     indexName,
```

Comment:
We won't need the indexName to be passed into the update function anymore. We can pull it off of the `alert` returned from the `fetchAlert` instance via the `_index` property.